build/cmake/lib/webview_chromium/CMakeLists.txt

```diff
@@ -62,8 +70,149 @@ if(NOT CEF_ROOT)
     if(result)
         message(FATAL_ERROR "Build step for cef failed: ${result}")
     endif()
+endfunction()
+
+if(NOT CEF_ROOT)
+    include(cef_version_info.cmake)
+
+    # Note that CMAKE_OSX_ARCHITECTURES is always defined under macOS, but is
+    # empty if not explicitly specified, so do _not_ use if(DEFINED ...) here.
+    if(CMAKE_OSX_ARCHITECTURES)
+        foreach(osx_arch IN LISTS CMAKE_OSX_ARCHITECTURES)
+            # Determine the matching CEF platform name.
+            if(${osx_arch} STREQUAL "arm64")
+                set(CEF_PLATFORM "macosarm64")
+            elseif(${osx_arch} STREQUAL "x86_64")
+                set(CEF_PLATFORM "macosx64")
+            else()
+                message(FATAL_ERROR "Building CEF for macOS architecture ${osx_arch} is not supported")
+            endif()
+
+            if(NOT first_cef_platform)
+                # Download/unpack CEF files in the root directory and remember
+                # that we did it by setting this variable.
+                set(first_cef_platform ${CEF_PLATFORM})
+            else()
+                list(APPEND other_cef_platforms ${CEF_PLATFORM})
+
+                # Use different subdirectories for the other architectures.
+                set(CEF_DOWNLOAD_DIR "${CEF_DOWNLOAD_ROOT}/${CEF_PLATFORM}")
+                set(CEF_SOURCE_DIR "${CEF_SOURCE_ROOT}/${CEF_PLATFORM}")
+                set(CEF_BUILD_DIR "${CEF_BUILD_ROOT}/${CEF_PLATFORM}")
+            endif()
+
+            wx_download_cef(${CEF_PLATFORM})
+        endforeach()
+
+        # Now lipo all the binaries together unless this is a degenerate case
+        # in which CMAKE_OSX_ARCHITECTURES contains only one element.
+        if(other_cef_platforms)
+            set(cef_binaries
+                "cef_sandbox.a"
```

Comment:
Is `cef_sandbox.a` needed to lipo? Because we turn the sandbox off.

Reply:
We don't use it currently but it seems better/safer to lipo all the binaries just in case...